### ISSUE TITLE
Installation failure

### ISSUE THREAD

**Issue by svewap** [mageplus/mageplus#16]
Mage+ can't be installed. After entering DB access data I get the following error:

```
a:5:{i:0;s:238:"Error in file: "/..../mageplus/app/code/core/Mage/Sales/sql/sales_setup/upgrade-1.6.0.3-1.6.0.4.php" - SQLSTATE[42000]: Syntax error or access violation: 1049 Unknown database 'sales_order_aggregated_created'";i:1;s:1130:"#0 /..../mageplus/app/code/core/Mage/Core/Model/Resource/Setup.php(645): Mage::exception('Mage_Core', 'Error in file: ...')
#1 /..../mageplus/app/code/core/Mage/Core/Model/Resource/Setup.php(437): Mage_Core_Model_Resource_Setup->_modifyResourceDb('upgrade', '1.6.0.3', '1.6.1.0')
#2 /.../mageplus/app/code/core/Mage/Core/Model/Resource/Setup.php(320): Mage_Core_Model_Resource_Setup->_upgradeResourceDb('1.6.0.3', '1.6.1.0')
#3 /.../mageplus/app/code/core/Mage/Core/Model/Resource/Setup.php(235): Mage_Core_Model_Resource_Setup->applyUpdates()
#4 /.../mageplus/app/code/core/Mage/Core/Model/App.php(417): Mage_Core_Model_Resource_Setup::applyAllUpdates()
#5 /.../mageplus/app/code/core/Mage/Core/Model/App.php(343): Mage_Core_Model_App->_initModules()
#6 /.../mageplus/app/Mage.php(684): Mage_Core_Model_App->run(Array)
#7 /.../mageplus/index.php(92): Mage::run('', 'store')
#8 {main}";s:3:"url";s:1:"/";s:11:"script_name";s:10:"/index.php";s:4:"skin";s:7:"default";}
```

**Comment by leesbian** [comment]
fixed in commit 39e49b5679ff5d0523a306ac48f20a8970907970

**Comment by svewap** [comment]
Thank you!
